From 03a8bf4e9c2a2971d0208840592892e4b5854b2e Mon Sep 17 00:00:00 2001
From: Terrence Pu <terrence.pu@amlogic.com>
Date: Tue, 25 May 2021 12:33:14 -0700
Subject: [PATCH 6/6] RDK: disable continuously print out from drm

PD#SWPL-

Problem:
continuously drm "add mode" debug print can cause scripting via console failed.

Solution:
change some drm debug print to higher level

Verify:
local

Change-Id: I30bb3ec14d04d49296541d336313d07bd8205e49
Signed-off-by: Terrence Pu <terrence.pu@amlogic.com>
---
 drivers/amlogic/drm/meson_hdmi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/amlogic/drm/meson_hdmi.c b/drivers/amlogic/drm/meson_hdmi.c
index b467a5037d6b..7cd0b2bdea15 100644
--- a/drivers/amlogic/drm/meson_hdmi.c
+++ b/drivers/amlogic/drm/meson_hdmi.c
@@ -228,7 +228,7 @@ int am_hdmi_tx_get_modes(struct drm_connector *connector)
 
 			drm_mode_probed_add(connector, mode);
 
-			DRM_INFO("add mode [%s]\n", mode->name);
+			DRM_DEBUG("add mode [%s]\n", mode->name);
 		}
 
 		if (pref_mode)
@@ -277,7 +277,7 @@ static enum drm_connector_status am_hdmitx_connector_detect
 {
 	int hpdstat = drm_hdmitx_detect_hpd();
 
-	DRM_INFO("am_hdmi_connector_detect [%d]\n", hpdstat);
+	DRM_DEBUG("am_hdmi_connector_detect [%d]\n", hpdstat);
 	return hpdstat == 1 ?
 		connector_status_connected : connector_status_disconnected;
 }
-- 
2.31.1

